[2023-06-05T21:55:22.309+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: process_weather_people_postgres.group_users.get_1_user scheduled__2023-06-04T00:00:00+00:00 [queued]>
[2023-06-05T21:55:22.325+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: process_weather_people_postgres.group_users.get_1_user scheduled__2023-06-04T00:00:00+00:00 [queued]>
[2023-06-05T21:55:22.325+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2023-06-05T21:55:22.325+0000] {taskinstance.py:1363} INFO - Starting attempt 4 of 7
[2023-06-05T21:55:22.325+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2023-06-05T21:55:22.344+0000] {taskinstance.py:1383} INFO - Executing <Task(PythonOperator): group_users.get_1_user> on 2023-06-04 00:00:00+00:00
[2023-06-05T21:55:22.355+0000] {standard_task_runner.py:55} INFO - Started process 7341 to run task
[2023-06-05T21:55:22.361+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'process_weather_people_postgres', 'group_users.get_1_user', 'scheduled__2023-06-04T00:00:00+00:00', '--job-id', '89', '--raw', '--subdir', 'DAGS_FOLDER/process_weather_people_postgres/process_weather_people_postgres.py', '--cfg-path', '/tmp/tmphn9v28af']
[2023-06-05T21:55:22.363+0000] {standard_task_runner.py:83} INFO - Job 89: Subtask group_users.get_1_user
[2023-06-05T21:55:22.454+0000] {task_command.py:376} INFO - Running <TaskInstance: process_weather_people_postgres.group_users.get_1_user scheduled__2023-06-04T00:00:00+00:00 [running]> on host 1160da0dacdb
[2023-06-05T21:55:22.556+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=data_eng
AIRFLOW_CTX_DAG_ID=process_weather_people_postgres
AIRFLOW_CTX_TASK_ID=group_users.get_1_user
AIRFLOW_CTX_EXECUTION_DATE=2023-06-04T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=4
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-06-04T00:00:00+00:00
[2023-06-05T21:55:23.152+0000] {get_user.py:36} INFO - The file user_inputt_1.csv was generated successfully.
[2023-06-05T21:55:23.152+0000] {python.py:177} INFO - Done. Returned value was: None
[2023-06-05T21:55:23.164+0000] {taskinstance.py:1406} INFO - Marking task as SUCCESS. dag_id=process_weather_people_postgres, task_id=group_users.get_1_user, execution_date=20230604T000000, start_date=20230605T215522, end_date=20230605T215523
[2023-06-05T21:55:23.216+0000] {local_task_job.py:164} INFO - Task exited with return code 0
[2023-06-05T21:55:23.253+0000] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
